clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -546,6 +562,7 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
     }
     return true;
   }
+
```

Comment:
```suggestion
```

Reply:
This one must reside there.
It splits function body from variable-member of class.